astropy/coordinates/representation.py

```diff
@@ -223,6 +224,27 @@ def _unitstr(self):
                            for component in self.components]))
         return unitstr
 
+    def __eq__(self, other):
+        """
+        Check for equality with another representation.
+        """
+        try:
+            comparisons = [getattr(self, component) ==
+                           getattr(other, component)
+                           for component in self.components]
+        except:
```

Comment:
Am I misunderstanding your comment? I don't want you to raise an Exception I meant something like `except Exception` if you don't know what Exceptions might occur. This prevents python from catching Segfaults, MemoryErrors and Keyboardinterrupts.